bai-tap-buoi-10/main.js

```diff
@@ -0,0 +1,140 @@
+//Bài 1:
+for(let i = 1 ; i <= 10; i++) {
+  for (let j = 1; j <= 10; j++){
+    let sum = 0
+    sum = i * j;
+    console.log(i + ' * ' + j + ' = ' + sum);
+  }
+}
+
+//Bài 2:
+const getNumberEven = number => {
+  if(number >= 1 && number <= 30){
+    for (let i = 1; i <= number; i++){
+      if (i % 2 == 0){
```

Comment:
```suggestion
      if (i % 2 === 0) {
```

Dùng `===` thay cho `==` để tăng tính chặt chẽ